eraser-map.yaml

```diff
@@ -919,7 +919,7 @@ layers:
         data: { source: osm, layer: water }
         draw:
             grid:
-                order: 5
+                order: 2
```

Comment:
I see! Looks like several tunnel rules have `order: 6`, but I would expect overlaps between tunnels and piers to be much less frequent than pier/water overlaps. I'll set water back to `order: 5` and piers to `order: 6`.